templates/bot/ts/workflow/src/adaptiveCards/doStuffActionResponse.json

```diff
@@ -0,0 +1,33 @@
+{
+  "type": "AdaptiveCard",
+  "body": [
+    {
+      "type": "TextBlock",
+      "size": "Medium",
+      "weight": "Bolder",
+      "text": "✅[ACK] ${title}"
+    },
+    {
+      "type": "TextBlock",
+      "text": "${body}",
+      "wrap": true
+    },
+    {
+      "type": "ActionSet",
+      "actions": [
+        {
+          "type": "Action.OpenUrl",
+          "title": "Bot Framework Docs",
+          "url": "https://docs.microsoft.com/en-us/azure/bot-service/?view=azure-bot-service-4.0"
+        },
+        {
+          "type": "Action.OpenUrl",
```

Comment:
I feel like this default action is too simple and would make it difficult to extend.. I suggest we can have an action that responds with another card so it showcases "workflows" are triggered by actions and steps are chained by responding card.

Reply:
If user want to extend the workflow, they can repeat the steps (1234) as showed in the `README`. For default template, I think just one action can demonstrate the code for card actions.